src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -231,6 +248,19 @@ private String getTokenToUse(String authTokenCredentialId, String token) {
         return token;
     }
 
+    private String correctEmojiFormat(String iconEmoji) {
+        if(StringUtils.isEmpty(iconEmoji)) {
+            return iconEmoji;
+        }
+        else if (!iconEmoji.startsWith(":")) {
```

Comment:
You could shorten this to
```suggestion
        iconEmoji = StringUtils.appendIfMissing(iconEmoji, ":")
```